components/dashboard/sections/platform-subscription/PlatformSubscriptionCard.tsx

```diff
@@ -0,0 +1,164 @@
+                      amount: (
+                        <FormattedMoneyAmount
+                          amount={props.subscription.plan.pricing.pricePerAdditionalCollective.valueInCents}
+                          currency={props.subscription.plan.pricing.pricePerAdditionalCollective.currency}
+                        />
+                      ),
+                    }}
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="flex gap-3">
+              <div>
+                <Receipt />
+              </div>
+              <div className="flex flex-col gap-1">
+                <div>
+                  <b>
+                    <FormattedMessage
+                      defaultMessage="{includedCount} Paid Expenses"
+                      id="V35NqH"
+                      values={{
+                        includedCount: props.subscription.plan.pricing.includedExpensesPerMonth,
+                      }}
+                    />
+                  </b>
+                  &nbsp;
+                  <Tooltip>
+                    <TooltipTrigger>
+                      <Info size={14} />
+                    </TooltipTrigger>
+                    <TooltipContent>
+                      <FormattedMessage defaultMessage="Completed expenses with the 'Paid' status." id="IHdyue" />
+                    </TooltipContent>
+                  </Tooltip>
+                </div>
+                <div className="text-sm text-muted-foreground">
+                  <FormattedMessage
+                    defaultMessage="{amount} / expense after that"
+                    id="F7fqJa"
+                    values={{
+                      amount: (
+                        <FormattedMoneyAmount
+                          amount={props.subscription.plan.pricing.pricePerAdditionalExpense.valueInCents}
+                          currency={props.subscription.plan.pricing.pricePerAdditionalExpense.currency}
+                        />
+                      ),
+                    }}
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        )}
+        {props.subscription.isCurrent && !props.subscription.endDate && (
+          <div className="w-full rounded-xl rounded-t-none bg-green-100 py-2 text-center text-sm font-bold text-oc-blue-tints-900">
+            <FormattedMessage
+              defaultMessage="On {dueDate}, your subscription will be renewed and you will be billed {basePrice}"
+              id="yrldr5"
+              values={{
+                dueDate: <FormattedDate dateStyle="medium" timeZone="UTC" value={props.billing.dueDate} />,
+                basePrice: (
+                  <FormattedMoneyAmount
+                    amount={props.billing.baseAmount.valueInCents}
+                    currency={props.billing.baseAmount.currency}
+                  />
+                ),
+              }}
+            />
+          </div>
+        )}
```

Comment:
It is weird that the total amount, including the prorated from a previous plan appears in the current plan card:
For example, in this downgrade, the amount displayed in the footer is inexplicable. There's no way to understand why the bill will be $409 given the informations displayed.
<img width="1311" height="310" alt="image" src="https://github.com/user-attachments/assets/e63c2993-0f83-4569-9202-22741921558d" />

Reply:
Design is iterating on it